### PR TITLE
Math Operators and Loops

### PR DESCRIPTION
**Pull request by riqbucks**

### CODE REVIEW

File: ericsmodule/src/main/java/sdoward/com/ericsmodule/BeerView.java [hunk 1]
```diff
@@ -5,4 +5,6 @@
 public interface BeerView {
     void showError(Throwable throwable);
     void displayBeers(List<Beer>beers);
+    void displayBeerCount(int beerCount);
```
Comment (sdoward) [inline ericsmodule/src/main/java/sdoward/com/ericsmodule/BeerView.java]: This should be `displayAverageBeerPercentage(double beerPercantage)`, beer percentage needs decimels so an int will not work

File: ericsmodule/src/main/java/sdoward/com/ericsmodule/MainActivity.java [hunk 2]
```diff
@@ -58,4 +61,6 @@ public void displayBeers(List<Beer> beers) {
         recyclerView.setAdapter(beerAdapter);
 
     }
+
+     public void displaybeerCount (int beerCount) { beerPercentageTextView.setText(String.valueOf(beerCount));}
```
Comment (sdoward) [inline ericsmodule/src/main/java/sdoward/com/ericsmodule/MainActivity.java]: you will have to modify this to match the interface when you change it

**Comment by sdoward** [comment]
You can also look at the exact lines of code I changed here https://github.com/sdoward/eric_learning/commit/1cd27742c4307f506fb7509ba1a9a2107c91c361

File: ericsmodule/src/main/java/sdoward/com/ericsmodule/MainActivity.java [hunk 3]
```diff
@@ -37,7 +40,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void setUpPresenter() {
         BeerService beerService = new Retrofit.Builder()
-                .baseUrl("https://api.punkapi.com/v2/beers")
+                .baseUrl("https://api.punkapi.com/")
```
[the remark this answers is not in the document]
Comment (riqbucks) [reply ericsmodule/src/main/java/sdoward/com/ericsmodule/MainActivity.java]: I don't remember changing it.. It must have been by mistake

File: ericsmodule/src/main/java/sdoward/com/ericsmodule/MainActivity.java (hunk 2 above)
Comment (riqbucks) [reply ericsmodule/src/main/java/sdoward/com/ericsmodule/MainActivity.java]: Got it

**Comment by sdoward** [comment]
Ok it's looking better. But you still need to add the functionality to calculate the average beer percentage 

This should be done around line 26 of `BeerPresenter` you need to put the for loop there

**Pull request closed without merging**